syntax = "proto3";

option java_multiple_files = true;
option java_package = "org.qsn.api.rpc.grpc.proto.entity.customized";

package org.qsn.protobuf;

message FileUploadTxDataDto {
    bytes fileHash = 1;
}
